plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverCommandLine.java

```diff
@@ -232,7 +234,15 @@ static boolean executeCommandLineCommands(@Nullable CommandLine commandLine, @Nu
 
     static CommandLine getCommandLine() {
         try {
-            return new DefaultParser().parse(ALL_OPTIONS, Platform.getApplicationArgs(), false);
+            List<String> applicationArgs = Arrays.stream(Platform.getApplicationArgs()).collect(Collectors.toList());
+            int index = applicationArgs.indexOf(CoreApplicationActivator.ARG_ECLIPSE_KEYRING);
+            if (index >= 0) {
+                applicationArgs.remove(index);
+                if (applicationArgs.size() > index) {
+                    applicationArgs.remove(index);
+                }
+            }
```

Comment:
Is two is the max number of eclipse keyring args that could be set? I mean shouldn't we do it in cycle?

Reply:
It can be specified only once as "-eclipse.keyring file-path"